iso4217parse/__init__.py

```diff
@@ -177,7 +177,8 @@ def by_symbol(symbol, country_code=None):
 
         if tmp_res:
             return tmp_res
-        return res
+        if not country_code:
```

Comment:
```suggestion
        if country_code is None:
```
This is more explicit: only if no `country_code` is provided, return the complete `res`.